x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -1320,4 +1349,204 @@ export class TelemetryReceiver implements ITelemetryReceiver {
     }
     return this._esClient;
   }
+
+  public async getIndices(): Promise<string[]> {
+    const es = this.esClient();
+
+    this.logger.l('Fetching indices');
+
+    const request: CatIndicesRequest = {
+      expand_wildcards: ['open', 'hidden'],
+      filter_path: ['index'],
+      format: 'json',
+    };
+
+    return es.cat
```

Comment:
The Elasticsearch docs recommend against using the _cat APIs for non-human consumption https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html These are meant as debugging tools for human operators.

Reply:
@rudolf I've just [moved](https://github.com/elastic/kibana/pull/194004/commits/a43ecb8c01bf5041e9a115b133998bcef006725d) from `_cat` to `es.indices` API